frontend/packages/noobaa-storage-plugin/src/constants/index.ts

```diff
@@ -4,3 +4,10 @@ export const BY_IOPS = 'I/O Operations';
 export const BY_LOGICAL_USAGE = 'Logical Usage';
 export const BY_PHYSICAL_VS_LOGICAL_USAGE = 'Physical Vs Logical Usage';
 export const BY_EGRESS = 'Egress';
+
+export const USAGE_LABELS = {
+  [BY_LOGICAL_USAGE]: 'Logical used capacity per account (in TiB)',
+  [BY_PHYSICAL_VS_LOGICAL_USAGE]: 'Physical vs. Logical used capacity (in TiB)',
+  [BY_EGRESS]: 'Egress Per Provider (in PiB)',
+  [BY_IOPS]: 'I/O Operations count (in thousands)',
+};
```

Comment:
The units in brackets needs to be dynamic based upon what data is plotted on y axis, so that the user can understand whether the 30B in y tick values is Billions or Bytes.
You can try re-using `maxUnit` value calculated for the y tick values in the current code.

Reply:
Acknowledged...